### DISCUSSION TITLE
Is supernet being deprecated?

### DISCUSSION THREAD

**Discussion by alfarkas** [0xPolygon/polygon-edge#2063] [General]
Like the title says, is supernet being deprecated?
A few days ago all the documentation about supernets disappeared all of the sudden, and docs changed url. The new documentation doesn't have any information about supernets, the same exact thing happened before with edge, suddenly the docs disappeared and then it was declared that edge was deprecated and supernets were the way to go.

**Comment by jastisriradheshyam** [comment]
They have renamed `supernets` as `CDK validium`.
If you try to visit https://github.com/0xPolygon/supernets2-node it will redirect to https://github.com/0xPolygon/cdk-validium-node

Not sure about edge.
